wormhole-connect/scripts/checkForeignAssetsConfig.ts

```diff
@@ -60,24 +60,39 @@ const checkEnvConfig = async (
             } catch (e: any) {
               if (
                 e?.message === '3104 RPC not configured' ||
-                e?.message === 'wormchain RPC not configured'
+                e?.message === 'wormchain RPC not configured' ||
```

Comment:
maybe you could move these checks out to a function to make evident why those errors got ignored and make it maintainable

```suggestion
function isWormchainError({ message }: Error) {
  const messages = ['3104 RPC not configured', 'wormchain RPC not configured', 'wormchain RPC not configured'];
  return messages.includes(message);
}
```